React/finger-exercises/redux/02 - react-redux/src/app/screens/App/components/ShoppingCart/index.js

```diff
@@ -41,10 +42,14 @@ class ShoppingCart extends PureComponent {
   }
 }
 
+const mapStateToProps = state => ({
```

Comment:
try using store, instead of state when you are referring to the general store of the app. Use state for the particular one that you can have in a class